--- src/components/free-for-charitys-tools-for-success-components/Tools-For-Businesses/index.tsx
+++ src/components/free-for-charitys-tools-for-success-components/Tools-For-Businesses/index.tsx
@@ -62,7 +62,7 @@
               </>
             }
             buttonText="Available Here"
-            buttonLink="http://fiverr.com/"
+            buttonLink="https://www.fiverr.com/"
             imageSrc="/Images/fiverr.webp" // 👈 image passed as prop
           />
